spec/support/shared_examples/request_method.rb

```diff
@@ -218,17 +218,41 @@
     end
   end
 
-  it 'handles requests with proxy' do
-    conn_options[:proxy] = 'http://google.co.uk'
+  context 'when a proxy is provided as option' do
+    before do
+      conn_options[:proxy] = 'http://google.co.uk'
```

Comment:
Maybe this value is a bit not-self-documenting? "Does Google act as a proxy? How?" the reader (me) might think.